cantusdb/cantus_pipeline.md

```diff
@@ -0,0 +1,24 @@
+# CantusDB flattening and json-ld structures
+
+> Summarize:    
+>   1. Reconcile the flattened csv dump
+>   2. With the output run `jsonld/generate_jsonld.py`(which also takes `jsonld/context.jsonld` as an input)
+
+I was provided with a csv file of 50 entries of CantusDB (see [here](https://github.com/malajvan/linkedmusic-datalake/blob/main/cantusdb/cantusdb_50_chants%20(1).csv))
```

Comment:
Just a note to either change these links or keep these in your `main` branch or they won't be available for future!